.devcontainer/server/Dockerfile

```diff
@@ -0,0 +1,21 @@
+# See here for image contents: https://github.com/microsoft/vscode-dev-containers/tree/v0.245.2/containers/go/.devcontainer/base.Dockerfile
+
+# [Choice] Go version (use -bullseye variants on local arm64/Apple Silicon): 1, 1.19, 1.18, 1-bullseye, 1.19-bullseye, 1.18-bullseye, 1-buster, 1.19-buster, 1.18-buster
+ARG VARIANT="1.19-bullseye"
```

Comment:
Default should be "1.19" or "1.19-buster" but if we're correctly mentioning in `devcontainer.json` then it doesn't really matter.